FirebaseAuth/Sources/Swift/Auth/Auth.swift

```diff
@@ -1170,7 +1162,7 @@ extension Auth: AuthInterop {
    asynchronously on the main thread in the future.
    */
   @available(iOS 13, tvOS 13, macOS 10.15, macCatalyst 13, watchOS 7, *)
-  @objc public func verifyPasswordResetCode(_ code: String) async throws -> String {
```

Comment:
Why does `@objc` go away?

Reply:
This is the async/await variation. objc only needs then non-async one.